Composer/packages/client/src/recoilModel/dispatchers/__tests__/project.test.tsx

```diff
@@ -68,12 +67,33 @@ jest.mock('../../parsers/lgWorker', () => {
   return {
     flush: () => new Promise((resolve) => resolve(null)),
     addProject: () => new Promise((resolve) => resolve(null)),
+
+    parseAll: (id, files) =>
+      new Promise((resolve) =>
+        files.map(({ id, content }) => {
+          require('@bfc/indexers').lgUtil.parse(id, content, files);
```

Comment:
Is there any reason we can't import this module?

Also, I don't see `resolve` ever getting invoked.

Reply:
This is used to mock the worker. I just use the utils to get the result without using the worker. Do you have any suggestions about this one?